gigareview/console/tests/functional/EMReportJobCest.php

```diff
@@ -13,33 +13,46 @@ public function _before(FunctionalTester $I)
+
         $I->runShellCommand("/usr/local/bin/php /app/yii_test manuscripts-q/run --verbose", false);
-        $I->canSeeInDatabase('manuscript', ["manuscript_number" => "GIGA-D-22-00054", "article_title" => "A machine learning framework for discovery and enrichment of metagenomics metadata from open access publications", "editorial_status" => "Final Decision Accept", "editorial_status_date" => "2022-06-07"]);
-        $I->canSeeInDatabase('manuscript', ["manuscript_number" => "GIGA-D-22-00060", "article_title" => "A chromosome-level genome of the booklouse, Liposcelis brunnea provides insight into louse evolution and environmental stress adaptation", "editorial_status" => "Final Decision Reject", "editorial_status_date" => "2022-06-07"]);
-        $I->canSeeInDatabase('manuscript', ["manuscript_number" => "GIGA-D-22-00030", "article_title" => "A novel ground truth multispectral image dataset with weight, anthocyanins and brix index measures of grape berries tested for its utility in machine learning pipelines", "editorial_status" => "Final Decision Pending", "editorial_status_date" => "2022-06-07"]);
+        $I->canSeeInDatabase("manuscript", ["manuscript_number" => "GIGA-D-22-00054", "article_title" => "A machine learning framework for discovery and enrichment of metagenomics metadata from open access publications", "editorial_status" => "Final Decision Accept", "editorial_status_date" => "2022-06-07"]);
+        $I->canSeeInDatabase("manuscript", ["manuscript_number" => "GIGA-D-22-00060", "article_title" => "A chromosome-level genome of the booklouse, Liposcelis brunnea provides insight into louse evolution and environmental stress adaptation", "editorial_status" => "Final Decision Reject", "editorial_status_date" => "2022-06-07"]);
+        $I->canSeeInDatabase("manuscript", ["manuscript_number" => "GIGA-D-22-00030", "article_title" => "A novel ground truth multispectral image dataset with weight, anthocyanins and brix index measures of grape berries tested for its utility in machine learning pipelines", "editorial_status" => "Final Decision Pending", "editorial_status_date" => "2022-06-07"]);
+        $I->canSeeInDatabase("ingest", ["file_name"=>"Report-GIGA-em-manuscripts-latest-214-20220607004243.csv", "report_type"=>1, "fetch_status"=>3, "parse_status"=>null, "store_status"=>null, "remote_file_status"=>null]);
+        $I->canSeeInDatabase("ingest", ["file_name"=>"Report-GIGA-em-manuscripts-latest-214-20220607004243.csv", "report_type"=>1, "fetch_status"=>3, "parse_status"=>1, "store_status"=>1, "remote_file_status"=>1]);
         $I->canSeeResultCodeIs(Exitcode::OK);
     }
 
-    public function tryToMatchManuscriptReportWithEntriesFoundInTable(FunctionalTester $I)
+    public function tryToMatchSampleManuscriptReportWithEntriesFoundInManuscriptTableAndUpdateStatusInIngestTable(FunctionalTester $I)
```

Comment:
It feels like this test is performing the same checks as in the previous `tryToPushManuscriptQueueJobToManuscriptTableAndUpdateStatusInIngestTable` test. The difference is that the first test uses hard-coded data to perform the checks, whereas this test uses data from `Report-GIGA-em-manuscripts-latest-214-20220607004243.csv` to perform the checks.

Reply:
This test aims to check the output of the manuscripts queue job is the same as the entries found in the input csv file.